Cargo.toml

```diff
@@ -64,6 +64,8 @@ serde_json = "1.0"
 smallvec = { version = "1.0", features = ["union"] }
 rustworkx-core = { path = "rustworkx-core", version = "=0.17.0" }
 flate2 = "1.0.35"
+pest = "2.7"
```

Comment:
By the way, changes to Cargo.toml should also have a corresponding Cargo.lock change. Did you forget to commit Cargo.lock?